.circleci/config.yml

```diff
@@ -17,3 +17,60 @@ jobs:
           if [ "${CIRCLE_BRANCH}" == "master" ]; then
             ./architect deploy
           fi
+
+  e2eSetup:
+    machine: true
+    steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./e2e-harness
+
+    - run: ./e2e-harness setup --name=ci-awsop-${CIRCLE_SHA1:0:7}
```

Comment:
What is the name used for? Should it be certop rather than awsop?

Reply:
This name is used for identifying the AWS resources created, very good catch, fixed.